bump-kernels

```diff
@@ -20,7 +20,7 @@ while [[ ${#} -gt 0 ]]; do
 			ekeyword ~all "${newk}"
 			"${scriptdir}"/copybump "${newk}"
 			${EDITOR:-vim} "${newk}"
-			GENTOO_MIRRORS= repoman manifest
+			pkgdev manifest
```

Comment:
We know the exact filename, so why not just use `ebuild ... manifest`?

Reply:
Sure.